bin/ghe-restore

```diff
@@ -448,8 +448,8 @@ else
   ghe-ssh "$GHE_HOSTNAME" -- /bin/sh 1>&3
 fi
 
-echo "Completed restore of $GHE_HOSTNAME from snapshot $GHE_RESTORE_SNAPSHOT"
+echo "Finished restore of $GHE_HOSTNAME from snapshot $GHE_RESTORE_SNAPSHOT"
 
 if ! $cluster; then
-  echo "Visit https://$hostname/setup/settings to review appliance configuration."
+  echo "To complete the restore process, please visit https://$hostname/setup/settings to review and save the appliance configuration."
```

Comment:
It should only be necessary to do this if you are restoring on to an unconfirmed appliance, as we automatically run `ghe-config-apply` on configured appliances at the end of the restore process: https://github.com/github/backup-utils/blob/7d0ca70cd3df6d1d0f3dd5d6ff6ded81a6db5c08/bin/ghe-restore#L400-L413

Maybe the `if` could be changed to `if ! $instance_configured; then` so that this message only appears on un-configured appliances.

Reply:
Ah, good point. Nice catch @snh.

Reply:
@keeran don't forget this 😉 (I'd like to merge this before merging by big PR so it makes it in before I create the legacy branch).

Reply:
@lildude ooops - read pre-coffee, thanks :)